specification/workloads/resource-manager/Microsoft.Workloads/preview/2021-12-01-preview/monitors.json

```diff
@@ -286,7 +286,12 @@
             "$ref": "#/parameters/monitorNameParameter"
           },
           {
-            "$ref": "#/parameters/TagsParameter"
```

Comment:
The previous swagger is really a bad design, every time you do the change like in this PR, it will be a breaking change for SDKs.